src/Controls/src/Core/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -71,5 +72,29 @@ public static partial class AppHostBuilderExtensions
 
 		public static IMauiHandlersCollection AddMauiControlsHandlers(this IMauiHandlersCollection handlersCollection)
 			=> handlersCollection.AddHandlers(DefaultMauiControlHandlers);
+
+		internal static MauiAppBuilder ConfigureImageSourceHandlers(this MauiAppBuilder builder)
+		{
```

Comment:
How, if at all, does this relate to AutomationProperties?

Reply:
This is just for code cleanup purposes. Putting everything in a better place.